### PR TITLE
Add theme toggle button

### PR DESCRIPTION
**Pull request by HNOONa-0**
# Description

I have added `theme` variable to grader store that way grader components can listen to changes made on it. I have added functions that load the `stylesheet` based on theme:

https://github.com/user-attachments/assets/5cfe9129-45fe-4627-9027-9fe467a4488c

Fixes: #7820

# Comments
I think I will undo slot change that we included in https://github.com/omegaup/omegaup/pull/7799 for two reasons:
1- I cant consistently find if the editor is full screen or not. For example, in this picture we can see that a slot value exists but it returns `undefined`, so I can decide if the editor is full screen or not based on slot value
Screenshot from 2024-09-14 22-59-36
2- I would like to add default values for `problem` prop for example. If we keep the slot, we have to include the same default values at multiple different components which is redundant.
# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.
- [x] The tests were executed and all of them passed.
- [ ] If you are creating a feature, the new tests were added.

### CODE REVIEW

File: frontend/www/js/omegaup/components/arena/EphemeralGrader.vue [hunk 1]
```diff
@@ -23,13 +23,21 @@ import Ephemeral from '../../graderv2/Ephemeral.vue';
...
+    default: () =>
+      Object.values(Util.supportedLanguages).map(
+        (languageInfo) => languageInfo.language,
+      ),
+  })
+  acceptedLanguages!: string[];
   @Prop({ default: 'cpp17-gcc' }) preferredLanguage!: string;
   @Prop({ default: true }) isEmbedded!: boolean;
-  @Prop({ default: 'vs' }) theme!: string;
+  @Prop({ default: Util.VS_LIGHT_THEME }) initialTheme!: string;
```
Comment (pabo99) [inline frontend/www/js/omegaup/components/arena/EphemeralGrader.vue:39]: Is it possible for the type of `initialTheme` to be an enum?

**Review by pabo99 — COMMENTED** [review]
Great job!! 

Just one minor change and for me it's okay

**Review by pabo99 — APPROVED** [review]
Thank you!

**Comment by pabo99** [comment]
@heduenas can you help us with the revision? This PR contains changes needed in other PRs. Thank you!

**Review by heduenas — APPROVED** [review]

**Pull request merged**